test/format/verify.c

```diff
@@ -137,12 +137,55 @@ table_mirror_fail_msg_flcs(WT_SESSION *session, const char *checkpoint, TABLE *b
       checkpoint ? " (checkpoint " : "", checkpoint ? checkpoint : "", checkpoint ? ")" : "");
 }
 
+/*
+ * position_cursor_before --
+ *     Place a cursor on the key directly preceding the target key. It's ok to return WT_NOTFOUND
+ *     from this function as any subsequent cursor next calls will begin at the start of the table.
+ */
+static int
+position_cursor_before(TABLE *table, WT_CURSOR *cursor, uint64_t target_keyno)
+{
+    WT_ITEM key;
+    int exact;
+    int ret;
+
+    key_gen_init(&key);
+    ret = 0;
+
+    switch (table->type) {
+    case FIX:
+    case VAR:
+        cursor->set_key(cursor, target_keyno);
+        break;
+    case ROW:
+        key_gen(table, &key, target_keyno);
+        cursor->set_key(cursor, &key);
+        break;
+    }
+
+    testutil_check(read_op(cursor, SEARCH_NEAR, &exact));
+
+    /* If we're on or past the target key then move backwards one key. */
+    if (exact >= 0) {
+        ret = read_op(cursor, PREV, NULL);
+        testutil_assert(ret == 0 || WT_NOTFOUND);
+        if (ret == WT_NOTFOUND)
+            ret = 0;
+    }
+
+    key_gen_teardown(&key);
+    return ret;
```

Comment:
```suggestion
    return (ret);
```

Reply:
Fixed in [d9fd6a8](https://github.com/wiredtiger/wiredtiger/pull/9660/commits/d9fd6a86fafaf9449bed957e42c84ccd3e114973)